File: database/enums.py
from enum import Enum

class AssetClass(Enum):
    STOCK = "Stock"
    BOND = "Bond"
    CRYPTO = "Crypto"
    ETF = "ETF"

class Side(Enum):
    BUY = "Buy"
    SELL = "Sell"

class Type(Enum):
    MARKET = "Market"
    LIMIT = "Limit"

class Status(Enum):
    NEW = "New"
    FILLED = "Filled"
    PARTIALLY_FILLED = "Partially Filled"
    REJECTED = "Rejected"

File: web/scripts/enums.js
const AssetClass = {
    Stock: 'Stock',
    Crypto: 'Crypto',
    ETF: 'ETF',
    Bond: 'Bond'
}

const Side = {
    Buy: 'Buy',
    Sell: 'Sell'
}

const Type = {
    Market: 'Market',
    Limit: 'Limit',
    Stop: 'Stop'
}
const Status = {
    New: 'New',
    PartiallyFilled: 'Partially Filled',
    Filled: 'Filled',
    Rejected: 'Rejected'
}

const Styles = {
    Green: 'green',
    Red: 'red'
}

const Endpoint = {
    Orders: '/orders',
    Positions: '/positions',
    Cash: '/cash',
    Performance: '/performance'
}

export { AssetClass, Side, Type, Status, Styles, Endpoint }